### PR TITLE
[Admin] Update link to fix build error

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[db27d30](https://github.com/OfficeDev/office-js-docs-pr/commits/db27d30351d0dbd67c8e4a2bb95a05c59938ffac)_: 

:clock10: **Pending**: waiting for processors ([5](https://ops.microsoft.com/#/statistic) builds ahead of you) 

 :warning: **Docs Build is busy**, currently there are 5 builds ahead of this one, for more information you can view the Build queue graph on the [Docs Portal](https://ops.microsoft.com/#/statistic).

**Comment by o365devx** [comment]
Docs Build status updates of commit _[db27d30](https://github.com/OfficeDev/office-js-docs-pr/commits/db27d30351d0dbd67c8e4a2bb95a05c59938ffac)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/referencing-the-javascript-api-for-office-library-from-its-cdn.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-fix-link/docs/develop/referencing-the-javascript-api-for-office-library-from-its-cdn.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/develop/referencing-the-javascript-api-for-office-library-from-its-cdn?branch=pr-en-us-1976) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C23%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202006231747505215-1976%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=3%2F0yMEfl%2BUUW0xlOTkeUGA62AF16ujdwXe49U1duams%3D&st=2020-06-23T17%3A44%3A24Z&se=2020-07-24T17%3A49%3A24Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[4c70dc5](https://github.com/OfficeDev/office-js-docs-pr/commits/4c70dc5952c1905a2cf5cddf168a2faac6b66f4e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/referencing-the-javascript-api-for-office-library-from-its-cdn.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-fix-link/docs/develop/referencing-the-javascript-api-for-office-library-from-its-cdn.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/develop/referencing-the-javascript-api-for-office-library-from-its-cdn?branch=pr-en-us-1976) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C23%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202006231803375901-1976%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Cbnv5GB0yaphTUsHR1EwPvK%2B%2BsyHeVxtneIgKfQfVWE%3D&st=2020-06-23T17%3A59%3A58Z&se=2020-07-24T18%3A04%3A58Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/develop/referencing-the-javascript-api-for-office-library-from-its-cdn.md [hunk 1]
```diff
@@ -30,24 +30,22 @@ If you plan to publish your Office Add-in from AppSource, you must use this CDN
...
 
 ```command&nbsp;line
 npm install --save-dev @types/office-js
 ```
 
 > [!NOTE]
-> To enable Intellisense for preview APIs, use the preview type definitions from [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/office-js-preview) by running the following command in the root of your project folder: 
->
-> `npm install --save-dev @types/office-js-preview`
+> To enable IntelliSense for preview APIs, see the [Preview](#preview) section later in this article.
```
Comment (AlexJerabek) [inline docs/develop/referencing-the-javascript-api-for-office-library-from-its-cdn.md:42]: I'm not sure this note is even needed anymore. It's right above the preview section, so readers may not know about "preview" until this point.

Comment (ElizabethSamuel-MSFT) [reply docs/develop/referencing-the-javascript-api-for-office-library-from-its-cdn.md:42]: makes sense - I'll remove that note

**Comment by o365devx** [comment]
Docs Build status updates of commit _[5bf4739](https://github.com/OfficeDev/office-js-docs-pr/commits/5bf473984a043bd098500e5fd0ea927ffd8c19ca)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/referencing-the-javascript-api-for-office-library-from-its-cdn.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-fix-link/docs/develop/referencing-the-javascript-api-for-office-library-from-its-cdn.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/develop/referencing-the-javascript-api-for-office-library-from-its-cdn?branch=pr-en-us-1976) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C23%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202006231818096465-1976%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=dsgHza6g%2FSAAtecnw0%2FWDy9S2voj4XcnYltrbPtbaZk%3D&st=2020-06-23T18%3A14%3A36Z&se=2020-07-24T18%3A19%3A36Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**